radis/io/hitran.py

```diff
@@ -1160,7 +1166,49 @@ def download_all_hitran_isotopes(molecule, directory):
                         )
                         os.remove(join(directory, file + ".data"))
                 try:
-                    fetch(file, get_molecule_identifier(molecule), iso, wmin, wmax)
+                      
+                    if(len(extra_params) != 0):
+                        fetch(file, get_molecule_identifier(molecule), iso, wmin, wmax, Parameters=extra_params) #fetching .data and .header file having extra params (comma separated in .data)
+                        
+                        from hapi import getColumn
+                        import numpy.ma as ma
+                        import pandas as pd
+                    
+                    
+                        file_params = {}  
+                        temp_dict = {}
+                        local_df = pd.DataFrame()
+                        global temp_df
```

Comment:
global parameter is very bad practice in Python, find another way ?

Reply:
Yes, I've removed the use of global df in my latest commit